data/templates/css/wikimedia.scss

```diff
@@ -572,4 +572,8 @@ $spacing-elements: 30px;
             text-align: left;
         }
     }
+    html {
+        -webkit-text-size-adjust:none;
+        font-size:100%
```

Comment:
Space between style prop name and :

Reply:
If possible, might be nice to have a comment here as to why we are explicitly setting `font-size` to `100%`